synthwave-theme.el

```diff
@@ -1,139 +1,400 @@
+   `(ac-completion-face ((,class ,completion-inline)))
+   `(ac-selection-face ((,class ,completion-selected-candidate)))
+
+   `(company-preview-common ((,class ,completion-inline)))
+   `(company-scrollbar-bg ((,class (:background ,synth-light-gray))))
+   `(company-scrollbar-fg ((,class (:background ,synth-pale-gray)))) ; Scrollbar (visible).
+   `(company-tooltip ((,class ,completion-other-candidates))) ; Suffix in tooltip.
+   `(company-tooltip-annotation ((,class (:weight normal :foreground ,synth-blue)))) ; Annotation.
+   `(company-tooltip-annotation-selection ((,class (:weight normal :foreground ,synth-light-gold)))) ; Annotation (for selection).
+   `(company-tooltip-common ((,class (:weight normal :foreground ,synth-pink :inherit company-tooltip)))) ; Prefix + common part in tooltip.
+   `(company-tooltip-common-selection ((,class (:weight normal :foreground ,synth-light-gold :inherit company-tooltip-selection)))) ; Prefix + common part in tooltip (for selection).
+   `(company-tooltip-selection ((,class ,completion-selected-candidate))) ; Suffix in tooltip (for selection).
+
+   `(compare-windows ((,class (:background ,synth-gold))))
+
+   `(compilation-error ((,class (:weight bold :foreground ,synth-deep-red)))) ; Used for grep error messages.
+   `(compilation-info ((,class ,grep-file-name)))
+   `(compilation-line-number ((,class ,grep-line-number)))
+   `(compilation-mode-line-exit ((,class (:weight bold :foreground ,synth-pale-green)))) ; :exit[matched]
+   `(compilation-mode-line-fail ((,class (:weight bold :foreground ,synth-lilac)))) ; :exit[no match]
+   `(compilation-mode-line-run ((,class (:weight bold :foreground ,synth-dark-gold)))) ; :run
+   `(compilation-warning ((,class (:weight bold :foreground ,synth-orange))))
+   `(popup-scroll-bar-background-face ((,class (:background ,synth-light-gray))))
+   `(popup-scroll-bar-foreground-face ((,class (:background ,synth-pale-gray)))) ; Scrollbar (visible).
+
+   ;; Button and link faces.
+   `(link ((,class ,link)))
+   `(link-visited ((,class ,link-visited)))
+   `(button ((,class ,link)))
+   `(header-line ((,class (:box (:line-width 1 :color ,near-near-white) :foreground ,synth-deep-pink :background ,near-near-black))))
+
+   ;; Line numbers
+   `(linum ((,class (:background "black" :foreground ,synth-pink))))
+   `(line-number ((,class (:background "black" :foreground ,synth-pink))))
+
+   ;; mode line faces
+   `(mode-line ((,class (:weight bold :background ,mode-line-active :foreground ,near-black))))
+   `(mode-line-buffer-id ((,class (:weight bold :background ,mode-line-inactive :foreground ,near-black))))
+   `(mode-line-emphasis ((,class (:weight bold :foreground "white"))))
+   `(mode-line-highlight ((,class (:weight bold :foreground "white"))))
+   `(mode-line-inactive ((,class (:weight bold :background ,mode-line-inactive :foreground ,near-black))))
+
+   ;; font-lock faces
+   `(font-lock-builtin-face ((,class (:weight bold :foreground ,synth-pale-cyan))))
```

Comment:
Lightening up the background washes out many of the colors. The background should be light, but keywords darker. Changing font-lock-builtin-face (and strings) to be darker helps. I'm testing with "#6cddf1" though I don't know I'd call it pale cyan, as it's bluer than the whole cyan palette. Perhaps swap with synth-lighter-blue

Reply:
I honestly cannot see the difference between synth-light-cyan and "#6cddf1". I did NOT incorporate this change.